src/cpu/instruction.rs

```diff
@@ -10,7 +10,7 @@ pub enum BranchCondition {
 }
 
 #[derive(Debug, Copy, Clone)]
-pub enum Instruction {
+pub enum Instruction8Bit {
```

Comment:
Should we do this, or just have `Instruction` and `ExtendedInstruction` or something

Reply:
nvm